packages/amplication-client/src/models.ts

```diff
@@ -592,6 +592,12 @@ export enum EnumAuthProviderType {
   Jwt = 'Jwt'
 }
 
+export enum EnumDbType {
+  mongo = 'mongo',
+  postgres = 'postgres',
+  mysql = 'mysql'
+}
+
```

Comment:
Did you add it manually?

Reply:
Our models are generated automatically. If you want this enum only in the client, please add it where you need in another file.
If you also need it for the server, please have a look at EnumGitProvider for example (in amplication-server).
After adding it to the server, run `npx nx build amplication-server` and you will see the changes in all models